ibis/backends/pandas/execution/generic.py

```diff
@@ -1066,7 +1065,7 @@ def execute_node_where_scalar_scalar_series(op, cond, true, false, **kwargs):
     return pd.Series(np.repeat(true, len(false))) if cond else false
 
 
-@execute_node.register(PandasTable, PandasBackend)
+@execute_node.register(ops.DatabaseTable, PandasBackend)
```

Comment:
We can't remove this until 4.0, since it's a breaking change.

Reply:
Right, then a `PandasInMemoryTable` could be an alternative